lib/dealership.rb

```diff
@@ -0,0 +1,54 @@
+class Dealership
+  attr_reader :inventory
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+    @inventory_count = 0
+    @has_inventory = false
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def inventory_count
+    @inventory.size
+  end
+
+  def has_inventory?
+    @has_inventory
```

Comment:
So we are getting our inventory count in the method above. So here, how would we know if we had inventory? What if we checked that our `inventory_count > 0`??